src/components/storyUtils/TextFieldShowcase/TextFieldShowcase.tsx

```diff
@@ -5,23 +5,32 @@ import TextField from 'components/TextField';
 import { Props as TextFieldProps } from 'components/TextField/TextField';
 
 type Props = {
+  disabled?: boolean;
   errorMsg?: React.ReactNode | string;
   label?: string;
+  lean?: boolean;
   status?: string;
+  styleType?: 'filled' | 'outlined' | 'elevated';
```

Comment:
i suggest move those to a global type as it will be used from the select, datepicker etc etc

Reply:
@panvourtsis I will add it as a type in `palette` file if you 're ok with it!